app/services/integrations/aggregator/taxes/invoices/payloads/avalara.rb

```diff
@@ -72,6 +72,14 @@ def fee_item(fee)
             def empty_struct
               @empty_struct ||= OpenStruct.new
             end
+
+            def item_amount(fee)
+              amount = fee.sub_total_excluding_taxes_amount_cents&.to_i&.fdiv(fee.amount.currency.subunit_to_unit)
+
+              amount *= -1 if invoice.voided?
```

Comment:
```suggestion
              amount &&= -amount if invoice.voided?
```

or just this if amount always present....

`amount = -amount if invoice.voided?`

is this any clearer?